enable/tests/test_assistant_test_case.py

```diff
@@ -23,7 +24,24 @@ def test_mouse_move():
     assert not event.shift_down
     nose.tools.assert_equal(event.window.get_pointer_position(), (10, 20))
 
-@skip_if_null
+
+def test_mouse_down():
+    test_assistant = EnableTestAssistant()
+    component = Component(bounds=[100, 200])
+    component.normal_left_down = mock.Mock()
+    test_assistant.mouse_down(component, x=0, y=0)
+    component.normal_left_down.assert_called_once()
+
+
+def test_mouse_dclick():
+    test_assistant = EnableTestAssistant()
+    component = Component(bounds=[100, 200])
+    component.normal_left_dclick = mock.Mock()
+    test_assistant.mouse_down(component, x=0, y=0, dclick=True)
```

Comment:
Instead of adding a flag for `mouse_down`, what do you think about having a separate method altogether?

Part of that is because Mouse Down != Mouse Click. Mouse Click includes both pressing the mouse down and releasing it at the same place. With mouse down and mouse release separated, one can, for example, test pressing the mouse somewhere, hold it, and then release it elsewhere. This would be what happens in a drag-and-drop action.
Double clicking a mouse key is a very different command: It involves system settings e.g. how quick two successive clicks should be considered as one double click event.

Reply:
Yea, that makes sense.